browser/tests/shortcut-rxjs.js

```diff
@@ -1,14 +0,0 @@
-require('../../register/rxjs');
-
-var assert = require('assert');
-var RxJsObservable = require('rxjs/Observable').Observable;
-var AnyObservable = require('../../');
-var implementation = require('../../implementation');
-
-it('rxjs-min', function () {
-	assert.strictEqual(AnyObservable, RxJsObservable);
-	assert.strictEqual(implementation, 'rxjs/Observable');
-	assert.strictEqual(typeof RxJsObservable.of, 'function');
-	assert.strictEqual(typeof RxJsObservable.from, 'function');
-	assert.strictEqual(typeof RxJsObservable.prototype.map, 'undefined');
-});
```

Comment:
They were mocha tests for the browser, and basically duplicates of existing AVA tests for node. With `karma-ava` we get code reuse if the `node` tests.

Also, testing this with `karma-ava` is so much more pleasant. With `karma-mocha` I had to launch a separate karma / brower instance for each test to achieve isolation (isolation is necessary because you can only `register` once). `karma-ava` gives you per test isolation with iframes, which is awesome. The tests run dramatically faster.